src/mame/midway/omegrace.cpp

```diff
@@ -269,6 +267,7 @@ class omegrace_state : public driver_device
 
 	TIMER_CALLBACK_MEMBER(periodic_int);
 	uint8_t vg_go_r();
+	inline uint8_t encode_spinner(uint8_t data);
```

Comment:
Putting `inline` on the declaration can lead to various pitfalls. If you want to suppress export of the implementation, you can put `inline` there. Also, it doesn't depend on any class state, so it should be `static` if you want it to be a member fighting. Since it's a pure function, you could make it `constexpr` if you write it so it doesn't need to mutate variants.